src/main/kotlin/com/manimdsl/linearrepresentation/Objects.kt

```diff
@@ -13,17 +13,18 @@ sealed class MObject : ManimInstr() {
     abstract val shape: Shape
 }
 
-sealed class ShapeWithBoundary(open val uid: String) : MObject() {
```

Comment:
Keep the sealed class and get rid of the ShapeWithBoundary interface, I think it makes more sense for it to extend MObject() as they all have Shapes underneath